### PR TITLE
Lint progress.

### PR DESCRIPTION
**Pull request by nathanielmanistaatgoogle**

### CODE REVIEW

**Comment by nathanielmanistaatgoogle** [comment]
Fun fact: [apparently 1.8.0 is right around the corner?](https://media.readthedocs.org/pdf/pylint/latest/pylint.pdf)

File: tools/distrib/pylint_code.sh [hunk 1]
```diff
@@ -19,15 +19,16 @@ set -ex
...
+    'src/python/grpcio/grpc'
+    'src/python/grpcio_health_checking/grpc_health'
+    'src/python/grpcio_reflection/grpc_reflection'
 )
 
 VIRTUALENV=python_pylint_venv
 
 virtualenv $VIRTUALENV
 PYTHON=$(realpath $VIRTUALENV/bin/python)
+$PYTHON -m pip install --upgrade pip
```
Comment (mehrdada) [inline tools/distrib/pylint_code.sh]: Thinking about this again, perhaps we should pin the `pip` version?

Comment (nathanielmanistaatgoogle) [inline tools/distrib/pylint_code.sh]: Not a bad idea - but if we do that, we should do it for all our virtual environments throughout the codebase. Let's defer for a separate change?

**Comment by nathanielmanistaatgoogle** [comment]
Relevant tests pass; merging.

**Pull request merged**